res/sections/40-WirelessMac.tex

```diff
@@ -0,0 +1,110 @@
+\chapter{Wireless MAC}
```

Comment:
Chiamerei il capitolo come nelle slide: Wireless MAC and Transport Protocols Interference

Reply:
Done :call_me_hand: